tests/codec_multipart.cpp

```diff
@@ -0,0 +1,168 @@
+#include <catch.hpp>
+#include <zmq_addon.hpp>
+
+#ifdef ZMQ_HAS_RVALUE_REFS
```

Comment:
Do all of these tests require rvalue references?

Reply:
This should probably be ZMQ_CPP11.

Reply:
Thanks, I'll check this.